sdk/storage/azure_storage_blob/src/generated/mod.rs

```diff
@@ -5,8 +5,3 @@
 
 pub mod clients;
 pub mod models;
-pub use clients::{
-    AppendBlobClient, AppendBlobClientOptions, BlobClient, BlobClientOptions, BlobContainerClient,
-    BlobContainerClientOptions, BlobServiceClient, BlobServiceClientOptions, BlockBlobClient,
-    BlockBlobClientOptions, PageBlobClient, PageBlobClientOptions,
-};
```

Comment:
From my understanding, omitting this would prevent customers from directly importing and using clients and client options that we _don't_ want them to.